test/controllers/observer_controller_test.rb

```diff
@@ -2613,6 +2741,25 @@ def test_create_observation_strip_images
     assert_false(old_img2.reload.gps_stripped)
   end
 
+  def test_create_observation_pending_naming_notification
+    params = {
+      observation: {
+        when: Time.zone.now,
+        place_name: locations(:albion).name,
+        specimen: "0",
+        thumb_image_id: "0"
+      },
+      name: {},
+      vote: { value: "3" }
+    }
+    login("rolf")
+
+    ObserverController.any_instance.stubs(:unshown_notifications?).returns(true)
+    post(:create_observation, params: params)
```

Comment:
**Same questions about stubbing**
(I couldn't figure out another way to make `unshown_notifications?` return `true`.)
The object is to cover line 372 of CreateAndEditObservations:
```ruby
365   def redirect_to_next_page
366     if @observation.location.nil?
...
371     elsif unshown_notifications?(@user, :naming)
372       redirect_to(action: "show_notifications", id: @observation.id)
```